misc/eclipse_formatting_nowrap.xml

```diff
@@ -207,7 +207,7 @@
         <setting id="org.eclipse.jdt.core.formatter.alignment_for_resources_in_try" value="80"/>
         <setting id="org.eclipse.jdt.core.formatter.use_tabs_only_for_leading_indentations" value="false"/>
         <setting id="org.eclipse.jdt.core.formatter.parentheses_positions_in_try_clause" value="common_lines"/>
-        <setting id="org.eclipse.jdt.core.formatter.alignment_for_selector_in_method_invocation" value="82"/>
```

Comment:
what happens if you just remove this line (and use default value)?

Reply:
No idea, this is what Eclipse exported for me when I changed the option to the default.